src/rasenmaeher_api/db/enrollments.py

```diff
@@ -28,6 +28,7 @@ class EnrollmentPool(ORMBaseModel):  # pylint: disable=R0903
     owner = sa.Column(saUUID(), sa.ForeignKey(Person.pk), nullable=False)  # whos mainly responsible
     active = sa.Column(sa.Boolean(), nullable=False, default=True)
     extra = sa.Column(JSONB, nullable=False, server_default="{}")  # Passed on to the enrollments
+    invitecode = sa.Column(sa.String(), nullable=True)
```

Comment:
If we do lookups based on this it should be indexed and if we expect to only get one result per code it should also be unique

Reply:
This came in from firstuserapi_orm_rambo